### PR TITLE
Add auth app config

### PR DESCRIPTION
**Pull request by levithomason**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -60,14 +64,16 @@ const addLocation = (app, localPort) => {
   app.location = app.locations[env]
 }
 
-addLocation(config.unity, 3000)
-addLocation(config.funnelAdvice, 4000)
+addLocation(config.auth, 3000)
+addLocation(config.funnelAdvice, 3010)
+addLocation(config.unity, 3020)
```
Comment (levithomason) [inline src/index.js]: Following hub port numbering pattern.  Skipping by 10s in a safe range.

**Comment by jhabdas** [comment]
:boat:

**Comment by kyleturco** [comment]
🕵

File: src/index.js [hunk 2]
```diff
@@ -60,14 +64,16 @@ const addLocation = (app, localPort) => {
...
 
 // ------------------------------------
 // Favicon
 // ------------------------------------
 
-config.unity.favicon = `${config.unity.location.href}/favicon.png`
+config.auth.favicon = `${config.auth.location.href}/favicon.png`
```
Comment (davezuko) [inline src/index.js]: Curious, why are these needed?
